lib/index.js

```diff
@@ -2,6 +2,8 @@ var fs = require('fs'),
 	path = require('path'),
 	colors = require('colors'),
 	common = require('./common'),
+	// Windows PowerShell translates magenta to the same blue as the background. So use white, instead.
+	inputAndMaskColor = process.title.indexOf('PowerShell') >= 0 ? 'white' : 'magenta',
```

Comment:
It's crazy that `process.title` is "Windows PowerShell". In the normal command prompt, the value is "Command Prompt - node  test.js".

What do you think about adding `process.platform === 'win32'` to the check?